src/main/java/algorithms/leetcode/utility/LinkedListUtility.java

```diff
@@ -0,0 +1,40 @@
+package algorithms.leetcode.utility;
+
+public class LinkedListUtility {
```

Comment:
Ur thinking the linked list will always int. Bug that won't happen in real world. It can be anything. So you have to generalise these utility methods. If i give linkedlist of characters or doubles or booleans or objects etc ur code should work

Reply:
In all the leetcode problems I have seen the value of the linked list is of INT type. So I simply added it here. Will try to generalise the Utility method.